components/push/android/src/test/java/mozilla/appservices/push/PushTest.kt

```diff
@@ -1,277 +0,0 @@
-    senderId will trigger 'hard wired' returns. These will exercise the Kotlin calling code.
-    The Rust Cargo tests for the various push subcomponents do a more thorough test of the push
-    code. I'd love to find a way to set a 'test' flag that rust can pick up, and be able to inline
-    some of the mocking calls those use.
-     */
-    protected val mockSenderId = "test"
-
-    protected val vapidPubKey = "BBCcCWavxjfIyW6NRhqclO9IZj9oW1gFKUBSgwcigfNcpXSfRk5JQTOcahMLjzO" +
-            "1bkHMoiw4b6L7YTyF8foLEEU"
-
-    protected fun getPushManager(): PushManager {
-        val pm = PushManager(
-                senderId = mockSenderId,
-                bridgeType = BridgeType.TEST,
-                databasePath = dbFile
-        )
-        pm.update("TestRegistrationId")
-        return pm
-    }
-
-    /* Usage:
-
-        The push component is designed to be as light weight as possible. The "Push Manager"
-        handles subscription management and message decryption.
-
-        In general, usage would consist of calling:
-
-        ```kotlin
-        val manager = PushManager(
-            senderId = "SomeSenderIDValue",
-            bridgeType = BridgeType.FCM,
-            registrationId = systemProvidedRegistrationValue,
-            databasePath = "/path/to/database.sql"
-        )
-        val newEndpoints = manager.verifyConnection()
-        if newEndpoints.length() > 0 {
-            for (channelId in newEndpoints.keys()) {
-                // send the endpoint (newEndpoint[channelId]) to the process tied to channelId
-            }
-        }
-
-        // On new message:
-        // A new incoming message generally has the following format:
-        // {"chid": ChannelID, "body": Body, "con": Encoding, "enc": Salt, "crypto_key": DH}
-        val decryptedMessage = manager.decrypt(
-            channelID=message["chid"],
-            body=message["body"],
-            encoding=message["con"],
-            salt=message.getOrElse("enc", ""),
-            dh=message.getOrElse("crypto-key", "")
-        )
-
-        // On new subscription:
-        val subscriptionInfo = manager.subscribe(channelID, scope)
-            // the subscription info matches what is usually passed on to
-            // the requesting application. This could be JSON encoded and returned.
-
-        // On deleting a subscription:
-        manger.unsubscribe(channelID)
-            // returns true/false on server unsubscribe request. A False may cause a
-            // verifyConnection() failure and new endpoints generation
-
-        // On a new native OS registration ID change:
-        manager.update(newSubscriptionID)
-            // sets the new registration ID (sender ID) on the server. Returns a false if this
-            // operation fails. A failure may prevent future messages from being received.
-
-      ```
-    */
-
-    @Test
-    fun testAesgcmDecryption() {
```

Comment:
All those tests were moved into the Rust layer, they mostly were testing that UniFFI works - but I don't think we need such tests anymore!

We can re-add more tests at this layer if we have logic at this layer (for example, a new telemetry layer here, etc)